src/components/HorizontalCard.tsx

```diff
@@ -1,26 +1,6 @@
 import React, { ReactNode } from "react"
-import styled from "@emotion/styled"
 import Emoji from "./OldEmoji"
```

Comment:
We need to change this as well. We have a new `Emoji` component that is using Chakra.

Reply:
With that change, we will have to remove the support of `emojiSize` here. It should be renamed to `emojiFontSize` and we would need to update those props on the places where we are using it.

For example, in `pages/stablecoins.tsx`
```tsx
// before
<TokenCard
  key={idx}
  emoji={token.emoji}
  description={token.description}
  emojiSize={3}
/>

// now
<TokenCard
  key={idx}
  emoji={token.emoji}
  description={token.description}
  emojiFontSize="5xl"
/>
```

Reply:
Can you eleborate? What should I import?

Reply:
You can check that here in the Dependencies section https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md#update-dependencies

Basically you need to replace `OldEmoji` with the new emoji component located here: `src/components/Emoji`.

Reply:
Should I refactro `pages/stablecoins.tsx` also? Should I ask to get assigned for it in the issues section?